components/api-server/README.md

```
@@ -0,0 +1,11 @@
+# API Server
+
+The CLP API Server is a Rust-based HTTP server that exposes CLP functionality through REST APIs.
+
+## API reference
+
+All available API endpoints are defined in the [OpenAPI][openapi] Specification. The generated
+specification can be found [here][static-api-reference].
```

markdownlint-cli2 (0.23.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)